test/perf/perf_simple_query.cc

```diff
@@ -33,6 +33,19 @@
 #include "schema_builder.hh"
 #include "release.hh"
 #include <fstream>
+#include "alternator/executor.hh"
+#include "alternator/rmw_operation.hh"
+#include "alternator/tags_extension.hh"
+#include "utils/rjson.hh"
+#include "cql3/query_processor.hh"
+#include "service/storage_service.hh"
+#include "service/storage_proxy.hh"
+#include "cdc/log.hh"
+#include "cdc/metadata.hh"
+#include "cdc/generation.hh"
```

Comment:
Why do we have to include so much cruft? Is it because we have no equivalent to cql_test_env?

Maybe we should have an alternator_test_env.

Reply:
On the one hand, it does look like a use case for alternator_test_env. On the other, I don't want to encourage people to actually write any alternator unit tests with such env by providing a tool for it - everything aside microbenchmarks can and should be tested by the test/alternator pytest suite. But, on the third hand, I can just refuse to merge alternator unit tests which use this env if any appear in the future. I'll try extracting as many ugly details as possible to alternator_test_env and post v2.